qme/main/database/relational.py

```diff
@@ -114,7 +114,8 @@ def update_task(self, executor, updates=None):
 
     def get_task(self, taskid=None):
         """Get a task based on a taskid. Exits on error if doesn't exist. If
-           a task id is not provided, get the last run task.
+           a task id is not provided, get the last run task. Unlike the filesystem
+           database, this get_task supports getting a partial task id.
```

Comment:
Oi, I loved file database and glob is there, any reason to not have it uniform?

Reply:
I'm generally encouraging users to go for sqlite, so it's a bit of me being pushy :))

Reply:
okay, added it, begrudgingly :)